src/sentry/eventstream/kafka/protocol.py

```diff
@@ -33,23 +33,19 @@ def get_task_kwargs_for_insert(operation, event_data, task_state=None):
         # Rust (re)normalization here again would be too slow.
         event_data["data"] = EventDict(event_data["data"], skip_renormalization=True)
 
-        kwargs = {
-            "event": Event(
-                **{
-                    name: event_data[name]
-                    for name in [
-                        "group_id",
-                        "event_id",
-                        "project_id",
-                        "message",
-                        "platform",
-                        "datetime",
-                        "data",
-                    ]
-                }
-            ),
-            "primary_hash": event_data["primary_hash"],
-        }
+        event = SnubaEvent(
```

Comment:
What's the reason to make this a SnubaEvent instead of eventstore/model/Event ?

Reply:
It should eventually use eventstore/models/Event, I had this in mind as an intermediate step since we'll need further changes to that model to make this and the other non ingestion use cases work. I wanted to avoid making those other changes until after we've verified that the first part of that works properly (i.e. flipped the "store.use-django-event" event switch in prod.